omnigibson/examples/robots/curobo_example.py

```diff
@@ -0,0 +1,400 @@
+
+def q_to_command(q, robot):
+    # Convert target joint positions to command
+    command = []
+    for controller in robot.controllers.values():
+        command.append(q[controller.dof_idx])
+    command = th.cat(command, dim=0)
+    assert command.shape[0] == robot.action_dim
+    return command
+
+
+def test_curobo():
+    # Ask the user whether they want online object sampling or not
+    robot_options = ["R1", "Tiago"]
+    robot_type = choose_from_options(options=robot_options, name="robot options", random_selection=False)
+
+    robot_cfg = {
+        "type": robot_type,
+        "obs_modalities": "rgb",
+        "position": [0, 0, 0],
+        "orientation": [0, 0, 0, 1],
+        "self_collisions": True,
+        "action_normalize": False,
+        "grasping_mode": "assisted",
+        "controller_config": {
+            "base": {
+                "name": "JointController",
+                "motor_type": "position",
+                "command_input_limits": None,
+                "use_delta_commands": False,
+                "use_impedances": True,
+            },
+            "trunk": {
+                "name": "JointController",
+                "motor_type": "position",
+                "command_input_limits": None,
+                "use_delta_commands": False,
+                "use_impedances": True,
+            },
+            "arm_left": {
+                "name": "JointController",
+                "motor_type": "position",
+                "command_input_limits": None,
+                "use_delta_commands": False,
+                "use_impedances": True,
+                "pos_kp": 200.0,
+            },
+            "arm_right": {
+                "name": "JointController",
+                "motor_type": "position",
+                "command_input_limits": None,
+                "use_delta_commands": False,
+                "use_impedances": True,
+                "pos_kp": 200.0,
+            },
+            "gripper_left": {
+                "name": "JointController",
+                "motor_type": "position",
+                "command_input_limits": None,
+                "use_delta_commands": False,
+                "use_impedances": True,
+                "pos_kp": 1500.0,
+            },
+            "gripper_right": {
+                "name": "JointController",
+                "motor_type": "position",
+                "command_input_limits": None,
+                "use_delta_commands": False,
+                "use_impedances": True,
+                "pos_kp": 1500.0,
+            },
+        },
+    }
+    if robot_type == "Tiago":
+        robot_cfg["controller_config"]["camera"] = {
+            "name": "JointController",
+            "motor_type": "position",
+            "command_input_limits": None,
+            "use_delta_commands": False,
+            "use_impedances": True,
+        }
+
+    # Create env
+    cfg = {
+        "env": {
+            "action_frequency": 30,
+            "physics_frequency": 300,
```

Comment:
is this necessary?

Reply:
Not necessary, but having a higher frequency seems to help controllers to converge better.